slcshelterarrests/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java

```diff
@@ -99,24 +98,27 @@ public static void main( String[] args ) throws InterruptedException {
     // CUSTOM FUNCTIONS DEFINED BELOW
 
     public static String paddedTime( String time ) {
-      int original = Integer.parseInt(time);
-      String padded = String.format("%04d", original);
-      return padded;
+        int original = Integer.parseInt( time );
+        String padded = String.format( "%04d", original );
+        return padded;
     }
 
     public static String standardizeDateTime( Object myDate, Object myTime ) {
-      if (myDate != null && myTime == null) {
-        FormattedDateTime date = new FormattedDateTime( myDate.toString(), null, "MM/dd/yyyy", null);
-        return date.getDateTime();
-      }
-      if (myDate == null && myTime != null) {
-        FormattedDateTime time = new FormattedDateTime( null, paddedTime(myTime.toString()), null, "HHmm");
-        return time.getDateTime();
-      }
-      if (myDate != null && myTime != null) {
-        FormattedDateTime dateTime = new FormattedDateTime( myDate.toString(), paddedTime(myTime.toString()), "MM/dd/yyyy", "HHmm");
-        return dateTime.getDateTime();
-      }
-      return null;
+        if ( myDate != null && myTime == null ) {
```

Comment:
Got it. In that case, since `FormattedDateTime` is a utility class that will be reused across integration, it will need to be moved to some shared location so that it's not copied each time. I'll file a ticket for that.